### PR TITLE
Refactor docstrings in `post_processing.py`

### PR DESCRIPTION
**Pull request by yaseminbridges**
Refactoring docstrings for classes and functions in `post_processing.py` to follow google docstring fornatting

### CODE REVIEW

File: src/pheval/post_processing/post_processing.py [hunk 1]
```diff
@@ -44,7 +66,15 @@ def from_gene_result(pheval_gene_result: PhEvalGeneResult, rank: int):
 
 @dataclass
 class PhEvalVariantResult(PhEvalResult):
-    """Minimal data required from tool-specific output for variant prioritisation."""
+    """Minimal data required from tool-specific output for variant prioritisation
+    Args:
+        chromosome (str): The chromosome position of the variant
```
Comment (julesjacobsen) [inline src/pheval/post_processing/post_processing.py:71]: Does it matter if the chromosome is in UCSC e.g. `chr1` or INSDC format e.g. `1` what about sex and mitochondrial chromosomes?  Should these be numeric or letter forms e.g. `23`, `24`, `25` or `X`, `Y`, `M` and is `M` or `MT` required for mitochondrial? How will this affect the results if these are incorrect?

File: src/pheval/post_processing/post_processing.py [hunk 2]
```diff
@@ -76,7 +116,12 @@ def from_variant_result(pheval_variant_result: PhEvalVariantResult, rank: int):
 
 @dataclass
 class PhEvalDiseaseResult(PhEvalResult):
-    """Minimal data required from tool-specific output for disease prioritisation."""
+    """Minimal data required from tool-specific output for disease prioritisation
+    Args:
+        disease_name (str): Disease name for the result entry
+        disease_identifier (str): Disease identifier for the result entry
```
Comment (julesjacobsen) [inline src/pheval/post_processing/post_processing.py:122]: What namespace should the identifiers be in? OMIM, OrphaNet, ICD, MONDO...?

File: src/pheval/post_processing/post_processing.py [hunk 3]
```diff
@@ -18,7 +25,12 @@ class PhEvalResult:
 
 @dataclass
 class PhEvalGeneResult(PhEvalResult):
-    """Minimal data required from tool-specific output for gene prioritisation."""
+    """Minimal data required from tool-specific output for gene prioritisation result
+    Args:
+        gene_symbol (str): The gene symbol for the result entry
+        gene_identifier (str): The gene identifier for the result entry
```
Comment (julesjacobsen) [inline src/pheval/post_processing/post_processing.py:31]: What namespace should the identifiers be in? NCBI, Ensembl, HGNC?

**Comment by julesjacobsen** [comment]
This looks great and it massively improves the codebase. However it also highlights some issues regarding identifiers for chromosomes, genes and diseases. Can you add explanations about these as they will presumably affect the results to greater or lesser degrees.

**Comment by yaseminbridges** [comment]
> This looks great and it massively improves the codebase. However it also highlights some issues regarding identifiers for chromosomes, genes and diseases. Can you add explanations about these as they will presumably affect the results to greater or lesser degrees.

I've specified the identifiers for chromosomes, genes and diseases and also added in the Notes that these can ultimately be what is found in the phenopacket as it is used for matching purposes in the analysis

**Pull request merged**